modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/HorizontalCard.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayCardWithHorizontal} from '@clayui/card';
+import React, {useState} from 'react';
+
+export default function HorizontalCard({
+	actions = [],
+	cssClass,
+	disabled,
+	href,
+	icon,
+	inputName,
+	inputValue,
+	interactive,
+	locale: _locale,
+	onSelectChange = () => {},
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	selectable,
+	selected,
+	title,
+	...otherProps
+}) {
+	const [selectedValue, setSelectedValue] = useState(selected);
+
+	const handleSelectChange = (selected) => {
+		setSelectedValue(selected);
+
+		onSelectChange(selected);
+	};
+
+	return (
+		<ClayCardWithHorizontal
+			actions={actions}
+			checkboxProps={{
+				name: inputName,
+				value: inputValue,
+			}}
+			className={cssClass}
+			disabled={disabled}
+			dropDownTriggerProps={{
```

Comment:
To render tooltip on icon hover, matching the existing tag:
![screen](https://user-images.githubusercontent.com/5435511/97301958-b3f07100-1858-11eb-9d6e-4b744b720660.png)

Should this be a feature? I'm not sure, probably not.

Reply:
Don't know, but if we want to render a tooltip over the trigger (which I doubt) that wouldn't be the title of the card but something like `Actions`. I'd say to keep this simple for the moment just remove it and show no tooltip.

Reply:
Removing this would technically be a breaking feature, but one I honestly wouldn't mind doing. Chances seem to me extremely low that some customer has this tooltip, or its customization, as important part of their UX.

Reply:
If you want to keep it, taking a look to the [clay  v2 implementation](https://github.com/liferay/clay/blob/2.x/packages/clay-card/src/ClayHorizontalCard.soy#L126)  you can pass `actions` as the dropdown title

Reply:
In the end, I removed the tooltip. The change is basically a fix for a bug present in v2.